tailwind.config.js

```diff
@@ -4,7 +4,13 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {colors:{
+      'regal-blue': '#1D8BCC'
```

Comment:
Does the hex match the name "regal-blue" or should it be changed?

Reply:
the actual name is "Newport Blue", so i'll change "regal-blue" to "newport-blue"